src/components/Table/Table.tsx

```diff
@@ -17,6 +17,7 @@ export type Cell<T> = {
   content: number | string | ContentComponent<T>;
   colSpan?: number;
   type?: 'financial' | 'normal';
+  align?: 'left' | 'right';
```

Comment:
as this is align the proper way is to have a 'center' option as well

Reply:
I know but it is not a value we use in the table cell component before this addition and I don't want to add any more to this.